src/app/shared/services/blockchain/campaign.service.ts

```diff
@@ -204,6 +205,22 @@ export class CampaignService {
       this.errorService.handleError(),
     )
   }
+
+  private uploadMultipleDocuments(documents: File[] | undefined): Observable<IPFSDocument[]> {
+    if (documents && documents.length > 0) {
```

Comment:
```suggestion
    if (documents?.length > 0) {
```
Why not this? Is there any issue?

Reply:
Forgot about that feature for a moment 😄

Reply:
Actually, for some reason TypeScript is showing an error for `documents?.length` - error message is that `documents can be undefined`... I have no idea why.